WoWsShipBuilder.Web/Pages/ShipComparison.razor

```diff
@@ -167,127 +157,126 @@
-                    View acceleration
+                <MudButton FullWidth="true" DisableElevation="true" Variant="Variant.Outlined" Color="Color.Secondary" Size="Size.Small" OnClick="OpenAccelerationCharts" Disabled="@(ViewModel.SelectedShipList.Count == 0)">
+                    @Localizer.SimpleAppLocalization(nameof(Translation.ShipComparison_ViewAcceleration))
                 </MudButton>
             </MudStack>
-
-            <MudPaper Outlined="true" Square="true" Class="pa-3" Style="width: 100%">
-                <MudText GutterBottom="true" Style="font-size: large">
-                    Section
-                </MudText>
+            <MudPaper Outlined="true" Square="true" Class="pa-3 mb-3" Style="width: 100%">
                 <MudButtonGroup OverrideStyles="false" Class="gap-3" Style="flex-wrap: wrap" Size="Size.Small">
                     @foreach (var dataSection in ViewModel.DataSections)
                     {
                         bool isSelected = ViewModel.SelectedDataSection.Equals(dataSection);
                         <MudButton DisableElevation="true" Variant="Helpers.GetVariantFromBool(isSelected, Variant.Filled, Variant.Outlined)" Color="Helpers.GetColorFromBool(isSelected, Color.Primary, Color.Secondary)" OnClick="@(() => SelectDataSection(dataSection))">
-                            @ViewModel.ShipComparisonDataSectionToString(dataSection)
+                            @dataSection.LocalizeShipComparisonDataSection(Localizer)
                         </MudButton>
                     }
                 </MudButtonGroup>
             </MudPaper>
         </MudStack>
     </MudStack>
 
-    <MudDataGrid @ref="dataGrid" T="ShipComparisonDataWrapper" ServerData="@ReloadData" Groupable="false" SortMode="SortMode.Multiple" Filterable="false" ShowFilterIcons="true" FilterMode="DataGridFilterMode.Simple" Hover Outlined Bordered Dense Breakpoint="Breakpoint.None" Virtualize ColumnResizeMode="ResizeMode.Column" ShowColumnOptions="false">
-
+    <MudDataGrid @ref="dataGrid" T="GridDataWrapper" ServerData="@ReloadData" DragDropColumnReordering DragIndicatorSize="Size.Small" Groupable="false" SortMode="SortMode.Multiple" Hideable ShowColumnOptions="false" Filterable="false" ShowFilterIcons="false" FilterMode="DataGridFilterMode.Simple" Hover Outlined Bordered Dense Breakpoint="Breakpoint.None" Virtualize ColumnResizeMode="ResizeMode.Column">
         <ToolBarContent>
             <MudTextField T="string" ValueChanged="Search" Placeholder="@Localizer.GetAppLocalization(nameof(Translation.ShipSelectionWindow_SearchShip)).Localization" Adornment="Adornment.Start" AdornmentIcon="@Icons.Material.Filled.Search" IconSize="Size.Medium" Class="mt-n3" Clearable="true" Immediate="true" DebounceInterval="150"/>
-            <div style="width: 45%">
+            <div style="width: 5%">
```

Comment:
Instead of using divs with certain width styles, maybe it would be better to use a MudGrid?

Reply:
I tired with it but couldnt adjust the items the way i wanted so tried this other way and it worked